Polyfills/Canvas/Source/Canvas.h

```diff
@@ -68,6 +68,17 @@ namespace Babylon::Polyfills::Internal
         bool UpdateRenderTarget();
         Babylon::Graphics::FrameBuffer& GetFrameBuffer() { return *m_frameBuffer; }
 
+        struct PoolBuffer
+        {
+            Graphics::FrameBuffer* frameBuffer;
+            bool isAvailable;
+        };
+        std::vector<PoolBuffer> mPoolBuffers;
+        void NativeCanvas::PoolInit(int nBuffers);
```

Comment:
`NativeCanvas::` not needed here. Maybe put everything pool related in its own class.